it/ch05.md

```diff
@@ -0,0 +1,77 @@
+
+## Prova - Controlla
+
+![controllo pagamento](media/black_prove-check.png)
+
+Il tab `Prova/controlla` fornisce streumenti per provare un pagamento o validare una prova di pagamento. Questo è necessario con Monero poiché questi dettagli non sono disponibili sulla blockchain.
+
+### Prova una Transazione
+
+Questa funzione genera una prova di avvenuto pagamento ad un certo indirizzo. Dovrai inserire le seguenti informazioni:
+
+**(1) ID Transazione:** Questo è l'ID del pagamento per il quale stai creando una prova. Puoi trovare i dettagli della transazione selezionando il tab `Storico` dal menu a sinistra. 
+**(2) Indirizzo:** Questo è l'indirizzo al quale stai dando la prova di avvenuto pagamento.
+**(3) Messaggio (opzionale):** Questo è un messaggio opzionale che sarà firmato con i dettagli della transazione. Se scegli di includere un messaggio, anche l'altra parte dovrà includere esattamente lo stesso messaggio al fine di verificare la tua prova.
+**(4) Genera:** Fai click qui una volta inseriti tutti i dettagli per generare la tua prova.
+&nbsp;
+
+Una guida dettagliata è disponibile su getmonero.org ['Come provare un pagamento'](https://getmonero.org/it/resources/user-guides/prove-payment.html)
+
+### Controlla una Transazione
+
+Questa funzione prova l'avvenuto pagamento. Dovrai inserire le seguenti informazioni:
+
+**(5) ID Transazione:** Questo è l'ID del pagamento che vuoi verificare.
+**(6) Indirizzo:** Questo è l'indirizzo di ricezione del pagamento che vuoi verificare.
+**(7) Messaggio (opzionale):** Questo è il messaggio opzione che potrebbe essere stato incluso all'interno della prova.
+**(8) Firma:** Questa è la firma generata per provare il pagamento.
+**(9) Controlla:** Fai click qui una volta inseriti tutti i dettagli per controllare la validità della prova di avvenuto pagamento.
+
+## Database degli anelli (RingDB) condiviso
+
+![database degli anelli condiviso](media/black_sharedringdb.png)
+
+Questo è un strumento avanzato che può essere usato per migliorare la privacy delle firme ad anello. Gli output usati nelle firme ad anello possono essere adattati per mitigare perdite di privacy quando viene usato un fork che riusa le chiavi o per evitare output che certamente non possono essere spesi in questa transazione.
+
+**(1) Nome del file di blackball:** Questo strumento applicherà la blackball ad output noti per essere stati spesi. Dopo aver eseguito monero-blockchain-blackball, importa il file risultante per evitare di usare questi output come esche nella costruzione delle firme ad anello. Questo file è salvato di default nella cartella .shared-ringdb.
+**(2) Applica blackball ad output:** Questa azione applicherà o rimuovera la blackball ad un singolo output scelto. Gli output vengono rappresentati con stringhe da 64 caratteri. L'output inserito in questo campo non sarà usato come esca nella firma ad anello costruita. Gli output rimossi dalla blackball potrebbero (ma non necessariamente saranno) usati come esche.
```

Comment:
`Gli output rimossi dalla blackball potrebbero (ma non necessariamente saranno) usati come esche.` to `Gli output rimossi dalla blackball potrebbero (ma non necessariamente) essere usati come esche.`